_posts/2024/2024-01-15-the-first-five-questions.md

```diff
@@ -0,0 +1,110 @@
+why large social structures tend to become hierarchical
+and how who's allowed to interrupt whom is tied to perceived social status.
+
+So:
+suppose we have six useful hours in a day of teaching and learning
+(i.e., eight hours minus setup, coffee breaks, lunch, and wrap-up).
+At 45 minutes per episode,
+that gives us eight episodes per day
+or sixteen episodes in total.
+What should the motivating questions for each episode be?
+I've listed four below,
+and I'd like to set up an hour-long call later this month
+for anyone who's interested in expanding on them,
+replacing them with better ones,
+or suggesting another approach entirely.
+If you'd like to take part,
+please [email me](mailto:{{site.author.email}}) and I'll send you an invitation.
+
+Why do we have so many meetings?
+:   Coordination costs rise as the square of the number of people,
+    so large groups must (a)
+    make work uniform
+    so that sets of people are effectively interchangeable (e.g., assembly lines and typing pools),
+    (b) create sub-groups and invest effort in coordinating their work,
+    or (c) allow sub-groups to work independently and pay the cost of duplicated or contradictory effort.
+
+Why do so many meetings suck?
+:   Most people don't seek to maximize profit, but to maximize social standing within their peer group.
+    Some people do this by being helpful, but others try to do it by establishing social dominance.
+    Their meetings turn into one-up displays,
+    which often take the form of interrupting people they perceive as having lower social status.
+    Social norms (such as meeting moderation rules) exist in part to constrain such behavior.
+
+How should software developers be assessed?
+:   Exams and performance reviews are facts of life in academia and industry.
+    They exist for good reasons—some people truly are more skilled or more productive than others—but
+    every quantitative metric for comparing people will inevitably be gamed,
+    while every qualitative metric is subject to various biases.
+    One response is to allow people to define their own victory conditions,
+    but that makes it difficult or impossible to compare people.
+
+Who owns the software I write?
+:   Depending on your circumstances,
+    the software you write may belong to your employer
+    even if you wrote it on your own time using your own computer
+    (because they provided the training you needed to write it),
+    or they may only have an interest in it if you used their physical resources,
+    or it may be entirely yours as long as you wrote it on your own time.
+    The answer depends on your venue's rules about intellectual property;
+    that concept is less than two hundred years old,
+    but has expanded steadily over the last century
+    via processes such as regulatory capture
+    in ways that further the interests of profit-driven corporations.
+
+Why do hairdressers have to be licensed but programmers don't?
```

Comment:
I am not sure if this is the case all over the world. For example, I have what we call here a professional registration. I have to register with the Colegio de ingenieros, licenciados y técnicos de la pampa and pay a fee every year. Without that I cannot practice. But that is not mandatory in every province. 
Your point is still valid. Why you choose the hairdresser as example? I can think also in truck or taxi drivers as another example that perhaps is a more world wide case.

Reply:
I think any profession that is (a) licensed and (b) seen as having lower social standing than programming could work here.